### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by sphh** [harmsm/PyCmdMessenger#26]
Hi Mike,

There is a stupid typo throughout your code: _recieve_ should actually be _receive_. Not that it changes the functionality of your code!!

Stephan

**Comment by harmsm** [comment]
Good catch.  That's a silly mental block of mine.  Unless I see the squiggly red spelling underline, I miss it.  Patch on the way.
